### PR TITLE
[adapters] Shorten CI

### PR DESCRIPTION
**Pull request by PatKamin**

### CODE REVIEW

**Comment by lukaszstolarczuk** [comment]
I was just about to do some shortening myself 😉 Can you perhaps include my commit here: https://github.com/lukaszstolarczuk/unified-runtime/commit/823d41d4cab279a66404399e84de0dae2870f922

This is what we discussed yesterday.

**Comment by PatKamin** [comment]
> I was just about to do some shortening myself 😉 Can you perhaps include my commit here: [lukaszstolarczuk@823d41d](https://github.com/lukaszstolarczuk/unified-runtime/commit/823d41d4cab279a66404399e84de0dae2870f922)
> 
> This is what we discussed yesterday.

Done. Modified commit msg a bit, moved the info about the target branch to the PR title.

**Review by lukaszstolarczuk — APPROVED** [review]
thx!

File: .github/workflows/coverity.yml [hunk 1]
```diff
@@ -31,20 +31,12 @@ jobs:
...
-      - name: Install apt packages
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y doxygen
-
       - name: Install pip packages
         run: pip install -r third_party/requirements.txt
 
       - name: Configure CMake
         run: cmake -B $WORKDIR/build -DUR_ENABLE_TRACING=ON -DUR_DEVELOPER_MODE=ON -DUR_BUILD_TESTS=ON -DUR_FORMAT_CPP_STYLE=ON -DUMF_ENABLE_POOL_TRACKING=ON
```
Comment (lukaszstolarczuk) [inline .github/workflows/coverity.yml:38]: `UR_FORMAT_CPP_STYLE=OFF`

**Review by lukaszstolarczuk — COMMENTED** [review]
are you planning on doing similar PR on main branch...? 😉

File: .github/workflows/coverity.yml (hunk 1 above)
Comment (PatKamin) [reply .github/workflows/coverity.yml]: Done

**Review by kswiecicki — APPROVED** [review]

**Review by pbalcer — APPROVED** [review]

**Comment by pbalcer** [comment]
This is still in draft, is that intentional?

**Pull request merged**